guides/GITHUB.md

```diff
@@ -0,0 +1,334 @@
+# Getting Started with GitHub
+
+What is GitHub, and how do you use it to interact with and contribute to SourceCred?
+This guide will give you an introduction to GitHub and SourceCred.
+
+## Why do we use GitHub?
+
+For now, think of GitHub as a place to collaborate. 
+If you had a bunch of documents that you wanted to collaborate on, what would you do?
+You might consider putting them in a shared file service like [Google Docs](https://docs.google.com/), or perhaps keeping them as Word documents on your local machine. 
+Although these platforms offer general collaborative editing capabilities, GitHub
+offers a bit more:
+
+### Permissionless collaboration
+
+Anyone can propose changes. You don't need to be given permission to view or edit
+a document, as is required by some of the platforms mentioned previously.
+
+### More advanced reviewing tools
+
+When you open up a Google Doc, you might be able to comment or write in revision mode, but
+when you do so, you are working with the entire document. GitHub allows
+review on the level of change. Along with this more detailed ability to edit,
+there is an integrated approval workflow, straightforward visualizations of what changed 
+(these are called "diffs"), and powerful automatic testing (called continuous integration or CI).
+
+### More detailed history
+
+You are going to have basic questions like:
+
+ - Who added this piece?
+ - When and why was this added?
+
+And GitHub has built-in tools to do this (one such action is called [git blame](https://help.github.com/en/github/managing-files-in-a-repository/tracking-changes-in-a-file).)
+
+### Easy for many people to collaborate at once
+
+You can work in your own "alternate reality" without interfering with other work, even simultaneously. When ready these alternatives can be merged back together.
+You can also work on changes out of order, although this is more of an advanced topic.
+
+### Web Interfaces
+
+By way of some helpful features, the text files that you are working on can be served as web pages
```

Comment:
What does 'can be served as web pages' mean?

Reply:
It means if you have content that can be rendered on a static web server (generally a *.html file) you can deploy it to GitHub pages (essentially choosing a branch/subfolder on your repo) and it will render as a website.

Reply:
How would you like this stated to be more clear?

Reply:
I'm not sure I understand your second description. Are you saying that what you're working on (on Github) can be turned into its own website? Or that you can preview what your work would look like on the webpage you're a part of?

Reply:
GitHub pages ->https://pages.github.com/ let's you turn a repository, e.g. here is one of mine -> https://github.com/vsoch/vsoch.github.io into a rendered website, e.g., here is the rendering of those files. https://vsoch.github.io. That is what I am referring to when I say serving web pages. The repository is serving static files to render on the web server that is provided by GitHub.

Reply:
The same is true for the docs repository here...

Reply:
Updated with example.